src/test/java/seedu/duke/flashutils/commands/AddCommandTest.java

```diff
@@ -6,8 +6,7 @@
 import seedu.duke.flashutils.utils.Parser;
 import seedu.duke.flashutils.utils.Storage;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
If I am not mistaken using * with imports does not pass stylecheck. I would say the previous version was better if it worked.